tests/configs/test-006-ch-upgrade-2.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       spec:
         containers:
         - name: clickhouse-pod
-          image: yandex/clickhouse-server:21.1.7.1
+          image: altinity/clickhouse-server:21.8.altinity_prestable
```

Comment:
There should be no dependencies on altinity ClickHouse packeges, especially prestable. Use community ones

Reply:
You mean yandex/clickhouse-server:21.8 ?